ddev/CHANGELOG.md

```diff
@@ -25,6 +25,7 @@
 * Fix Github API search query ([#15943](https://github.com/DataDog/integrations-core/pull/15943))
 * Do not modify the Agent build name if provided by the user when running the e2e environments ([#16052](https://github.com/DataDog/integrations-core/pull/16052))
 * Bump the Python version in the dependency provider when bumping the Python version ([#16070](https://github.com/DataDog/integrations-core/pull/16070))
+* Fix `ddev env start` to allow the use of stable releases ([#16077](https://github.com/DataDog/integrations-core/pull/16077))
```

Comment:
Make sure to move this to a separate file. It's already showing up in the wrong release 🙈

Reply:
Ahh I forgot!